src/main/java/com/kaziamyr/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,18 @@
 package com.kaziamyr.onlinebookstore.dto;
 
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
```

Comment:
isbn field has rules to check if it is valid, please create your custom annotation and validator or use annotation "@ISBN" that in org.hibernate.validator.constraints.ISBN